sdk/azcore/policy_logging.go

```diff
@@ -71,48 +75,26 @@ func (p *requestLogPolicy) Do(req *Request) (*Response, error) {
 	tryDuration := tryEnd.Sub(tryStart)
 	opDuration := tryEnd.Sub(opValues.start)
 
-	logClass := LogResponse // Default logging information
-
-	// If the response took too long, we'll upgrade to warning.
-	if p.options.LogWarningIfTryOverThreshold > 0 && tryDuration > p.options.LogWarningIfTryOverThreshold {
-		// Log a warning if the try duration exceeded the specified threshold
-		logClass = LogSlowResponse
-	}
-
-	if err == nil { // We got a response from the service
-		sc := response.StatusCode
-		// Promote to Error any 4xx (except those listed is an error) or any 5xx.
-		// For other status codes, we leave the level as is.
-		if ((sc >= 400 && sc <= 499) && sc != http.StatusNotFound && sc != http.StatusConflict && sc != http.StatusPreconditionFailed && sc != http.StatusRequestedRangeNotSatisfiable) || (sc >= 500 && sc <= 599) {
-			logClass = LogError
-		}
-	} else { // This error did not get an HTTP response from the service; upgrade the severity to Error
-		logClass = LogError
-	}
-
-	if Log().Should(logClass) {
+	if Log().Should(LogResponse) {
 		// We're going to log this; build the string to log
 		b := &bytes.Buffer{}
 		slow := ""
-		if p.options.LogWarningIfTryOverThreshold > 0 && tryDuration > p.options.LogWarningIfTryOverThreshold {
-			slow = fmt.Sprintf("[SLOW >%v]", p.options.LogWarningIfTryOverThreshold)
+		if p.options.LogSlowResponse > 0 && tryDuration > p.options.LogSlowResponse {
+			slow = fmt.Sprintf("[SLOW >%v]", p.options.LogSlowResponse)
 		}
 		fmt.Fprintf(b, "==> REQUEST/RESPONSE (Try=%d/%v%s, OpTime=%v) -- ", opValues.try, tryDuration, slow, opDuration)
 		if err != nil { // This HTTP request did not get a response from the service
 			fmt.Fprint(b, "REQUEST ERROR\n")
 		} else {
-			if logClass == LogError {
-				fmt.Fprint(b, "RESPONSE STATUS CODE ERROR\n")
-			} else {
-				fmt.Fprint(b, "RESPONSE SUCCESSFULLY RECEIVED\n")
-			}
+			fmt.Fprint(b, "RESPONSE SUCCESSFULLY RECEIVED\n")
```

Comment:
I think this is slightly misleading, maybe it could say: "RESPONSE RECEIVED", successfully seems misleading if there's an error status code.

Reply:
Fixed